src/components/Tab.js

```diff
@@ -58,8 +71,8 @@ module.exports = React.createClass({
           'ReactTabs__Tab',
           className,
           {
-            'ReactTabs__Tab--selected': selected,
-            'ReactTabs__Tab--disabled': disabled,
+            [`${activeTabClassName}`]: selected,
+            [`${disabledTabClassName}`]: disabled,
```

Comment:
I think we could just do `[activeTabClassName]: selected` without template string?